<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainForm</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <property name="horizontalSpacing">
     <number>4</number>
    </property>
    <property name="verticalSpacing">
     <number>11</number>
    </property>
    <item row="2" column="2">
     <widget class="QLabel" name="lb_caps">
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font:  14pt;
color: rgb(76, 155, 235);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="bt_enter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>246</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:enabled{
	background-color: white;
	border-style: solid;
	border-width: 2px;
	border-radius: 15px;
	border-color: rgb(1, 140, 234);

	font: 14pt;
	padding: 6px;
	color: rgb(1, 140, 234);
 }
 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }

QPushButton:hover{
 	background-color: rgb(246, 246, 246);
}


QPushButton:pressed {
     background-color: rgb(240, 240, 240);
 }</string>
      </property>
      <property name="text">
       <string>Вход</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="lb_layout">
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font:  14pt;
color: rgb(76, 155, 235);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLineEdit" name="le_login">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: white;

	border-style: solid;
	border-width: 10px;
	border-color: white;
	border-radius: 15px;

	font:  14pt;
	color: rgb(1, 140, 234);
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Имя пользователя</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLineEdit" name="le_pass">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: white;

	border-style: solid;
	border-width: 10px;
	border-color: white;
	border-radius: 15px;

	font:  14pt;
	color: rgb(1, 140, 234);
}</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>256</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Пароль</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="bt_help_password">
      <property name="minimumSize">
       <size>
        <width>246</width>
        <height>45</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:enabled{
	background-color: white;
	border-style: solid;
	border-width: 2px;
	border-radius: 15px;
	border-color: rgb(1, 140, 234);

	font: 14pt;
	padding: 6px;
	color: rgb(1, 140, 234);
 }
 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }

QPushButton:hover{
 	background-color: rgb(246, 246, 246);
}


QPushButton:pressed {
     background-color: rgb(240, 240, 240);
 }</string>
      </property>
      <property name="text">
       <string>Забыли пароль?</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
   <zorder>bt_help_password</zorder>
   <zorder>bt_enter</zorder>
   <zorder>lb_layout</zorder>
   <zorder>horizontalSpacer_2</zorder>
   <zorder>lb_caps</zorder>
   <zorder>le_pass</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>verticalSpacer_2</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>le_login</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
